docs/getting-started/delete-on-dockstore.rst

```diff
@@ -1,11 +1,11 @@
-Deleting an entry on Dockstore
+Deleting an Entry on Dockstore
 ===================================
 
 Tutorial Goals
 --------------
 
 -  Deleting an entry
--  Uninstalling the Dockstore GitHub App from the source GitHub repo
+-  Ensuring that the entry does not re-appear on Dockstore
 
 This tutorial walks through the process of deleting entries from Dockstore.
 This action is permanent and after you delete the workflow, it will no longer appear on Dockstore.
```

Comment:
Suggesting saying "entry" in "after you delete the workflow" instead of "workflow" to be more generic